start.py

```diff
@@ -179,6 +179,12 @@ def _print(message: str):
     ssl_cert_file = certifi.where()
     os.environ["SSL_CERT_FILE"] = ssl_cert_file
 
+if "--headless" in sys.argv:
+    os.environ["OPENPYPE_HEADLESS_MODE"] = "1"
+    sys.argv.remove("--headless")
+else:
+    if os.getenv("OPENPYPE_HEADLESS_MODE") != "1":
+        os.environ.pop("OPENPYPE_HEADLESS_MODE")
```

Comment:
This is crashing if `OPENPYPE_HEADLESS_MODE` is not set.
```suggestion
        os.environ.pop("OPENPYPE_HEADLESS_MODE", None)
```